src/WebJobs.Extensions.Twilio/WebJobs.Extensions.Twilio.csproj

```diff
@@ -22,7 +22,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.32" />
-    <PackageReference Include="Twilio" Version="5.6.3" />
+    <PackageReference Include="Twilio" Version="6.*" />
```

Comment:
Can you please update this to a specific version instead of a floating version here? If the intent is to support anything >6.0.0 (or 6.0.1 which is what is published), you can specify that here and that customers will be able to use a higher version in the project.

Reply:
Is that the best practice? I couldn't really find guidance on this for .NET libraries in general.

Reply:
Updated to 6.2.0